sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/package-info.java

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
 
 /**
- * This package contains model classes for Azure Key Vault Certificates
+ * Package containing the data models for CertificateClient. The key vault client performs cryptographic key operations
+ * and vault operations against the Key Vault service.
```

Comment:
Autorest will generate a package-info for each package (folder) it generates into. Since this PR now has Autorest generating some of the public models it will generate a package-info for the public models. A future release of Autorest will allow for package-info modifications to be retained.